03-Security-In-CICD/Readme.md

```diff
@@ -231,6 +238,86 @@ what we have seen in the case of ad hoc SVT runs:
 ![03_AzSDK_Logs](../Images/03_AzSDK_Logs.png)
 
 [Back to top...](Readme.md#contents)
+
+# Security Verification Tests (SVTs) in Jenkins pipeline
+	
+> Note : AzSDK Plugin requires PowerShell to be present on Jenkins Server. Therefore, the Plugin is currently supported for Windows machines only.	
```

Comment:
the Plugin  => the plugin

Reply:
Many places Plugin is with caps P, i think it should be small

Reply:
Resolved